Link_sinFade_Library.ino

```diff
@@ -37,10 +42,8 @@ void HSV_sinFade_Retain_1Z (float Steps, int fadeZone, float startHSV[], float H
 
   for (int Stepr = 0; Stepr <= Steps; Stepr++) {    ///////////// <<<<<<<<<<<<  START FADE
 
-    if (! (Stepr % 10) ) {          // modulo operator % returns the remainder, e.g.: 10 % 2 = 0. 10 % 3 = 1.
-      // FIXME: should this be a map()? looks like a 0..1
-      ValKnob = analogRead(BRIGHTNESS) / 1043.9 + 0.02;            // analogRead is slow, so do every 10th
-    }                                                              //<< Knob reads value 0 to 1023.
+    ValKnob = brightness();
```

Comment:
I put the "get brightness knob" code all in one place. See end of the other .ino file.